app/controllers/tag_controller.rb

```diff
@@ -141,6 +138,9 @@ def show
       @notes = nodes.where('node.nid NOT IN (?)', @qids) if @node_type == 'note'
       @questions = nodes.where('node.nid IN (?)', @qids) if @node_type == 'questions'
     end
+    
+    @pagy, nodes = pagy(nodes.order(order_by), items: 24)
```

Comment:
So I think we need to make a new line on 137:

`nodes = nodes.order(order_by)`

Then we can change this line to:

```suggestion
    @pagy, nodes = pagy(nodes, items: 24)
```

That way the sorting still happens above, but the pagination happens below. Sound good?